govim.go

```diff
@@ -204,6 +221,8 @@ func NewGovim(plug Plugin, in io.Reader, out io.Writer, log io.Writer) (Govim, e
 
 		callVimNextID: 1,
 		callbackResps: make(map[int]callback),
+
+		instanceID: newInstanceID(),
```

Comment:
I think we can just make this an atomic sequence number, no? We don't need a random element in there as far as I can tell?

Reply:
Good point. Then you also get glanceable ordering that could be useful in some edge cases.

Updated and added `#` as prefix to make it easy to filter with `grep`.